.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,45 @@
+
+name: deploy docs
+on:
+  release:
+    types: [published]
```

Comment:
This workflow only works when released by this part.

Reply:
what will happen if we release 2.17.0, ..., 2.17.4, 2.18.0 and then release 2.17.5? will 2.17.5 override 2.18.0 posting?fff